manubot/cite/tests/test_doi.py

```diff
@@ -42,7 +42,9 @@ def test_get_doi_csl_item_zotero():
     doi = "10.1038/ng.3834"
     csl_item = get_doi_csl_item_zotero(doi)
     assert isinstance(csl_item, dict)
-    csl_item["author"][9]["family"] == "GTEx Consortium"
+    # for an unknown reason, GTEx Consortium becomes the first author
```

Comment:
This is very strange, and I don't understand why this reference has multiple inconsistent versions of the metadata.